INSTALL.md

```diff
@@ -66,16 +66,25 @@ To built AMS with [PFA](https://github.com/flux-framework/PerfFlowAspect) suppor
 
 ```
 $ cd $CODE_ROOT/setup
+$ source setup_env_with_pfa.sh
```

Comment:
We do not distribute the "setup" scripts on the ams repository. Make sure you can install ams with pfa enabled using the spack recipe. Please delete the respective lines.

Reply:
Fixed.